pkgs/development/python-modules/plexapi/default.nix

```diff
@@ -1,21 +1,35 @@
-{ lib, buildPythonPackage, fetchFromGitHub, requests
-, tqdm, websocket_client, pytest, pillow, isPy27 }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, tqdm
+, websocket_client
+, pytestCheckHook
+, pillow
```

Comment:
```suggestion
```

Since we don't run tests these two can be dropped.

Reply:
Thanks, I missed to remove them after check if there are tests which could be run.